specs/_template/01_main.rst

```diff
@@ -0,0 +1,253 @@
+.. Getting Started
+
+.. Installation and IDE
+    To make your editing easier, we recommend to use VSCode editor with the extensions:
+    - reStructuredText (https://marketplace.visualstudio.com/items?itemName=lextudio.restructuredtext)
+    - Table Formatter (https://marketplace.visualstudio.com/items?itemName=shuworks.vscode-table-formatter)
+.. 
+    The prerequisites for those extensions are python3 and doc8 packcages. Install instructions are here:
+    - https://docs.python-guide.org/starting/install3/osx/ (install python and pip, nothing else)
+    - After this, use pip3 to install other prerequisites below:
+    - https://docs.restructuredtext.net/articles/prerequisites.html
+
+.. rst syntax quick reference
+    If you are used to markdown, then rst is going to be a bit strange at first.
+    rst is more powerful than markdown, and it is standardised.
+
+.. Where to get more info
+    - http://docutils.sourceforge.net/docs/user/rst/quickref.html
+    - http://docutils.sourceforge.net/docs/user/rst/quickstart.html
+    - http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html
+    - http://docutils.sourceforge.net/docs/ref/rst/directives.html
+
+.. <------- comments start with 2 dots
+
+.. Document Name
+    Note that the lines must be at least as long as the text
+    ====================== 
+    This is document name
+    ======================
+
+.. Headers
+    Just underline with any symbol like = or - or + or ~ or `
+    Be consistent - different symbols are taken as different header levels.
+    Examples:
+..
+    1st level header
+    ==================
+..
+    2nd level header
+    ------------------
+..
+    3rd level header
+    ~~~~~~~~~~~~~~~~~~~
+..
+    4th level header
+    ++++++++++++++++++
+
+.. Lists
+    1. first item
+    2. second item
+    #. auto-generated item number
+
+.. Unordered lists
+    * first level
+    <blank line>
+        - second level
+    <blank line>
+        + third level
+
+.. Code
+    .. code:: <language (optional)>
+    <blank line>
+        this is going to be code
+
+.. Links
+    `This is link text`_ <-------- undersocre at the end
+    .. _`This is  link text`: https://example.org <------------- this is link definition
+    .. anchor: <---------- this is anchor to the text below it
+    `anchor`_ <----------- this is link to the anchor
+    `Some text <anchor_>`_ <----- text is different from the anchor name
+    We put all link definitions (except from the page info header table) to the end of the document.
+
```

Comment:
Most of this information is in the links provided. I would not put it here (which means more maintenance for us)